### PR TITLE
feat: implement previous abductions screen(PRLC100-156)

### PR DESCRIPTION
**Pull request by rahulsanjava19**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/app/case/case.ts [hunk 1]
```diff
@@ -212,6 +212,9 @@ export interface Case {
   op_courtProceedingsOrders?: C100OrderTypes[];
   op_otherProceedings?: OtherProceedings;
   c1A_safteyConcerns?: C1ASafteyConcerns;
+  c1A_previousAbductionsShortDesc?: string;
```
Comment (channa-patil) [inline src/main/app/case/case.ts:215]: @niranjan-301  Are we using these fields in any decision making or navigation ?
If not please remove them, they will added to userCase in session by default.

File: src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/content.test.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  caption: 'Safety concerns',
+  title: 'Provide details of the previous abductions',
+  line1: 'Give a short description of the previous incidents of abduction.',
+  c1A_previousAbductionsShortDescHint: 'Include any previous attempts to threaten or abduct the children.',
+  c1A_policeOrInvestigatorInvolved: 'Were the police, private investigators or any other organisation involved?',
+  c1A_policeOrInvestigatorInvolvedHint: 'Including in the UK or overseas.',
+  one: 'Yes',
+  two: 'No',
+  otherDetails: 'Provide more details',
+  guidanceParentalChildHyperlink: 'https://www.gov.uk/government/collections/child-abduction',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/content.test.ts:20]: @niranjan-301 Let's remove these common links & use it from common files present in abduction.

File: src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/content.ts [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  caption: 'Safety concerns',
+  title: 'Provide details of the previous abductions',
+  line1: 'Give a short description of the previous incidents of abduction.',
+  c1A_previousAbductionsShortDescHint: 'Include any previous attempts to threaten or abduct the children.',
+  c1A_policeOrInvestigatorInvolved: 'Were the police, private investigators or any other organisation involved?',
+  c1A_policeOrInvestigatorInvolvedHint: 'Including in the UK or overseas.',
+  one: 'Yes',
+  two: 'No',
+  otherDetails: 'Provide more details',
+  guidanceParentalChildHyperlink: 'https://www.gov.uk/government/collections/child-abduction',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/content.ts:17]: Please remove these links.

File: src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/template.njk [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{% extends "common/template.njk" %}
+
+
+{% block page_content %}
+<span class="govuk-caption-xl">{{caption}}</span>
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{title}}</h1>
+<p class="govuk-body-m govuk-!-margin-bottom-1">{{line1}}</p>
+{% endblock %}
+{% block page_safetyConcerns %}
+<p class="govuk-body-m">
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/template.njk:10]: @niranjan-301 Please remove these lines & include common template.njk file in abduction folder.

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/app/case/case.ts (hunk 1 above)
Comment (rahulsanjava19) [reply src/main/app/case/case.ts]: done

File: src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/content.test.ts (hunk 2 above)
Comment (rahulsanjava19) [reply src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/content.test.ts]: done

File: src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/template.njk (hunk 4 above)
Comment (rahulsanjava19) [reply src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/template.njk]: done

**Review by channa-patil — APPROVED** [review]

**Pull request merged**